http_propagation.go

```diff
@@ -36,47 +36,48 @@ type HTTPFormat struct{}
 
 var _ propagation.HTTPFormat = (*HTTPFormat)(nil)
 
-func parseHeader(h string) (trace.SpanContext, bool) {
+// ParseAmazonHeader parses an Amazon trace header to OpenCensus span context.
+func ParseAmazonHeader(header string) (trace.SpanContext, bool) {
```

Comment:
do you think it's necessary to have "Amazon" in the name since it is already implied by the package? I think it's fine but it does seem slightly redundant.

Reply:
Done, named it ParseTraceHeader to associate it with tracing.